rezervo/providers/ibooking/provider.py

```diff
@@ -313,3 +313,15 @@ async def find_public_ibooking_class(
             err.log("Schedule get request denied")
             return BookingError.ERROR
         return find_class_in_schedule_by_config(_class_config, schedule)
+
+    async def verify_authentication(self, credentials: ChainUserCredentials) -> bool:
+        async with ClientSession() as session:
+            auth_session = (
+                await authenticate_session(
+                    session, credentials.username, credentials.password
+                ),
+            )
+            return not isinstance(
+                auth_session[0],
```

Comment:
For some reason, this is a tuple, not a Union type in runtime

Reply:
The extra comma in line 322 makes it a [single element tuple](https://docs.python.org/3/tutorial/datastructures.html#tuples-and-sequences) :upside_down_face: 
```
auth_session = (
    (...),
)
```